app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 4)) == 17, \
+        "sum of returned list should be equal value"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(30, 6) == [5, 5, 5, 5, 5, 5], \
+        "value should be split on equal parts"
```

Comment:
You are creating an unnecessary variable in your assert statement. You should directly compare the result of the function call to the expected result. Refer to the Code Style section of the checklist.